tests/testthat/helper-model.R

```diff
@@ -84,11 +84,15 @@ expect_categorical_vars_works <- function(model) {
 }
 
 expect_can_tune_boost_tree <- function(model) {
```

Comment:
I don't see how this tests the multi-predict feature. Don't we want to test something like `predict(model, data, n_trees = c(1,2,3))`

Reply:
I added extra tests for multi_predict